pages/Configuring/Uncommon-tips-&-tricks.md

```diff
@@ -114,6 +114,62 @@ bind = $mod, S, movetoworkspace, special:magic
 bind = $mod, S, togglespecialworkspace, magic
 ```
 
+## Show desktop
+
+This approach uses same principle as the [Minimize windows using special workspaces](#minimize-windows-using-special-workspaces) section.
+It moves all windows from current workspace to a special workspace named `desktop`.
+Showing desktop state is remembered per workspace.
+
+Create a script:
+
+```sh
+#!/bin/env sh
+
+TMP_FILE="$XDG_RUNTIME_DIR/hyprland-show-desktop"
+
+CURRENT_WORKSPACE=$(hyprctl monitors -j | jq '.[] | .activeWorkspace | .name' | sed 's/"//g')
+
+if [ -s "$TMP_FILE-$CURRENT_WORKSPACE" ]; then
+  readarray -d $'\n' -t ADDRESS_ARRAY <<< $(< "$TMP_FILE-$CURRENT_WORKSPACE")
+
+  for address in "${ADDRESS_ARRAY[@]}"
+  do
+    CMDS+=" dispatch movetoworkspacesilent name:$CURRENT_WORKSPACE,address:$address;"
+  done
+
+  hyprctl --batch$CMDS
```

Comment:
Same here, missed this.

Reply:
Everything should be solved with last commit.

Reply:
The quotation marks _must_ be present, or the command will break.